test/test_dht.cpp

```diff
@@ -2583,11 +2594,11 @@ TORRENT_TEST(traversal_done)
 
 	// invert the ith most significant byte so that the test nodes are
 	// progressively closer to the target item
-	for (int i = 0; i < num_test_nodes; ++i)
+	for (std::size_t i = 0; i < num_test_nodes; ++i)
 		nodes[i].id[i] = ~nodes[i].id[i];
 
 	// add the first k nodes to the subject's routing table
-	for (int i = 0; i < 8; ++i)
+	for (std::size_t i = 0; i < 8; ++i)
 		t.dht_node.m_table.add_node(nodes[i]);
```

Comment:
could this be a range-for over ``nodes``?

Reply:
can't modify this one because nodes is of `9` elements, but using it in others